packages/components/src/select/tests/chromatic/SelectGhost.stories.tsx

```diff
@@ -0,0 +1,555 @@
+            <Item key="mars">Mars</Item>
+            <Item key="mercury">Mercury</Item>
+            <Item key="neptune">Neptune</Item>
+            <Item key="pluto">Pluto</Item>
+            <Item key="saturn">Saturn</Item>
+            <Item key="uranus">Uranus</Item>
+            <Item key="venus">Venus</Item>
+        </Select>
+    )
+};
+
+export const ScrollingRelectedItemOutsideScope: SelectStory = {
+    name: "scrolling with selected item outside initial visible scope",
+    render: () => (
+        <Select defaultSelectedKey="venus" defaultOpen placeholder="Select a planet" aria-label="Planets" variant="ghost">
+            <Item key="ceres">Ceres</Item>
+            <Item key="charon">Charon</Item>
+            <Item key="earth">Earth</Item>
+            <Item key="eris">Eris</Item>
+            <Item key="jupiter">Jupiter</Item>
+            <Item key="haumea">Haumea</Item>
+            <Item key="makemake">Makemake</Item>
+            <Item key="mars">Mars</Item>
+            <Item key="mercury">Mercury</Item>
+            <Item key="neptune">Neptune</Item>
+            <Item key="pluto">Pluto</Item>
+            <Item key="saturn">Saturn</Item>
+            <Item key="uranus">Uranus</Item>
+            <Item key="venus">Venus</Item>
+        </Select>
+    )
+};
+
+export const CustomTriggerWidth: SelectStory = {
+    name: "custom trigger width",
+    render: () => (
+        <Select width="20rem" defaultOpen placeholder="Select a planet" aria-label="Planets" variant="ghost">
+            <Item key="earth">Earth</Item>
+            <Item key="mars">Mars</Item>
+            <Item key="saturn">Saturn</Item>
+        </Select>
+    )
+};
+
+export const CustomMenuWidth: SelectStory = {
+    name: "custom menu width",
+    render: () => (
+        <Select overlayProps={{ width: "20rem" }} defaultOpen placeholder="Select a planet" aria-label="Planets" variant="ghost">
+            <Item key="earth">Earth</Item>
+            <Item key="mars">Mars</Item>
+            <Item key="saturn">Saturn</Item>
+        </Select>
+    )
+};
+
+export const Direction: SelectStory = {
+    name: "direction",
+    render: () => (
+        <Select direction={{ md: "top", lg: "bottom" }} defaultOpen placeholder="Select a planet" aria-label="Planets" variant="ghost">
+            <Item key="earth">Earth</Item>
+            <Item key="mars">Mars</Item>
+            <Item key="saturn">Saturn</Item>
+        </Select>
+    )
+};
+
+export const DirectionBottom: SelectStory = {
+    name: "direction bottom",
+    render: () => (
+        <Select direction="bottom" defaultOpen placeholder="Select a planet" aria-label="Planets" variant="ghost">
+            <Item key="earth">Earth</Item>
+            <Item key="mars">Mars</Item>
+            <Item key="saturn">Saturn</Item>
+        </Select>
+    )
+};
+
+export const DirectionTop: SelectStory = {
+    name: "direction top",
+    render: () => (
+        <Div paddingTop="100px">
```

Comment:
This used to be a chromaticCanvas Prop, as I could not find a way to do it in CSF3, might be a me issue, I went this way.